README.md

```diff
@@ -6,7 +6,9 @@ Thunderstore is a mod database and API for downloading mods.
 
 ## Setup guide for development
 
--   Copy `.env.template` to `.env` and modify as you see fit
+-   Copy `.env.template` to `.env` and modify as you see fit - If you have access to the `python-packages` submodule & it's cloned, make
+    sure to set `BUILD_INSTALL_EXTRAS` to `true`. Any other value will not work. Changing this setting will require
```

Comment:
1. Could the value for `BUILD_INSTALL_EXTRAS` be `True` so it would match the other boolean values in the `.env.template`?
2. If the consumer requires lowercase, could we somehow still support both values in the env file?
3. If that's not possible, maybe document the supported value as a comment in the file (it might be hard to find the info in this README)

Reply:
My point was to make things more developer-friendly, but sure, let's serve our machine overlords instead :trollface:

Reply:
I'd have done it if it wasn't painful to do a similar bool check with shell scripts as python has, but unfortunately that's not the case so this is what we'll get to deal with for now.